CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/parser/PCAPImportFileParser.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.importexport.delimited.parser;
+
+import au.gov.asd.tac.constellation.plugins.parameters.PluginParameters;
+import io.pkts.PacketHandler;
+import io.pkts.Pcap;
+import io.pkts.buffer.Buffer;
+import io.pkts.packet.Packet;
+import io.pkts.packet.IPPacket;
+import io.pkts.packet.TCPPacket;
+import io.pkts.packet.TransportPacket;
+import io.pkts.packet.UDPPacket;
+import io.pkts.protocol.Protocol;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.openide.util.Exceptions;
+import org.openide.util.lookup.ServiceProvider;
+
+
+/**
+ *
+ * @author serpens24
+ */
+@ServiceProvider(service = ImportFileParser.class)
+public class PCAPImportFileParser extends ImportFileParser {
+
+    private static final Logger LOGGER = Logger.getLogger(PCAPImportFileParser.class.getName());    
+    private ArrayList<String[]> allResults = new ArrayList<>();
```

Comment:
I would declare this as simply a List and only instantiate as an ArrayList
i.e. `private List<String[]> allResults = new ArrayList<>();`

Reply:
done - and moved into the specific method that uses it.